frontend-project/src/pages/institutions/new/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import { Button, Col, Card, Form, Input, Row, Select } from 'antd';
+import React from 'react';
+import { formatMessage, FormattedMessage } from 'umi-plugin-react/locale';
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 16 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const InstitutionsNewForm: React.FunctionComponent = () => {
```

Comment:
jak zmienisz import na ten co dałem, to będzie można zrobić
```ts
const InstitutionsNewForm: FunctionComponent = () => {
```

Reply:
wypchnięte

Reply:
Jak dla mnie jest ok, ale jak pisałem nie chcę zatwierdzać jak jest jeszcze magul przypisany do review